mmaction/core/evaluation/__init__.py

```diff
@@ -1,17 +1,23 @@
-from .accuracy import (average_precision_at_temporal_iou,
-                       average_recall_at_avg_proposals, confusion_matrix,
-                       get_weighted_score, interpolated_precision_recall,
-                       mean_average_precision, mean_class_accuracy,
-                       mmit_mean_average_precision, pairwise_temporal_iou,
-                       softmax, top_k_accuracy)
+from .accuracy import (MeanClsAcc, TopKAcc, confusion_matrix,
+                       get_weighted_score, mean_class_accuracy, top_k_accuracy)
```

Comment:
Seems TopKAcc and top_k_accuracy are duplicated now, while there is no way for a user to use top_k_accuracy

Reply:
`TopKAcc` with `top_k_accuracy` is similar with `BinaryLogisticRegressionLoss` with `binary_logistic_regression_loss`. If users want to use the function, they can directly import by `from mmaction.core import top_k_accuracy`.

Maybe I didn't get your meaning

Reply:
I mean, we used to set `--eval top_k_accuracy` for testing. Now that we refactored the metrics, some docs may need to be changed (`--eval top_k_accuracy`  doesn't work now, since `top_k_accuracy` is not in allowed_metrics). 
And part of my last comments are not right: top_k_accuracy and TopKAcc are not duplicated.

Reply:
Maybe we can keep some old command like `--eval top_k_accuracy` for several versions but raise a deprecated warning until we totally remove them.

Reply:
Right, so you may need to modify some codes in evaluate functions of datasets